docs/developers/guides/gas/gas-fees.mdx

```diff
@@ -36,12 +36,73 @@ and gas used.
 `linea_estimateGas` is the recommended method for estimating gas on Linea. See our 
 [reference page](../../reference/api/linea-estimategas.mdx) for more information. 
 
-Linea also supports [`eth_estimateGas`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_estimategas),
-[`eth_gasPrice`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_gasprice), and
-[`eth_feeHistory`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_feehistory).
+Linea also supports:
+- [`eth_estimateGas`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_estimategas)
+- [`eth_gasPrice`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_gasprice)
+- [`eth_feeHistory`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_feehistory).
+
+## Gas pricing
+
+The gas price you retrieve using methods such as `linea_estimateGas` is based on the variable data
+cost of the previous block with a profitability multiplier applied.
+
+The first stage of returning gas price values with `linea_estimateGas` is to populate values in an
+object called `extraData`. The two most relevant values here are:
+- A `FIXED_COST` of 0.03 Gwei, which reflects infrastructure costs, and;
+- `VARIABLE_COST`, which is the cost per byte of data submitted to L1. 
+
+These values are inserted into each block's `extraData` field—32 bytes available in every Ethereum 
+block to store any data—and then used by the sequencer and Linea Besu nodes running the correct 
+plugins to expose `linea_estimateGas`, in both cases with the same logic.
```

Comment:
Recommend (assuming we're writing to Linea's `extraData`):

The gas price returned by [`linea_estimateGas`](link) is based on the
variable data cost of the previous block with a profitability multiplier applied.

Each Linea block's `extraData` field is populated with the following gas price values that are used by linea_estimateGas to calculate the cost of a transaction:

- A FIXED_COST of 0.03 Gwei, which reflects infrastructure costs, and;
- VARIABLE_COST, which is the cost per byte of data submitted to L1.

:::note
The `extraData` field is a 32-byte space used to store arbitrary data, such as metadata or
additional information relevant to the block.

With Linea it is used by the sequencer and [Linea Besu nodes](link) running the correct plugins
to expose `linea_estimateGas`
:::

Reply:
Strictly speaking, there's a third field, which is used to set the recommended `eth_gasPrice` response. Because vanilla `eth_gasPrice` response may yield values that will make a lot of transactions not includable